spec/models/plan_spec.rb

```diff
@@ -880,18 +900,53 @@
       end
     end
 
-    it "when user is a reviewer" do
+    it "when user is a reviewer and feedback requested" do
       # All reviewers should be able to comment
-      subject.roles.reviewer.pluck(:user_id).each do |user_id|
-        expect(subject.commentable_by?(user_id)).to eql(true)
-      end
+      user = create(:user)
```

Comment:
not a big deal, but you could offload these duplicated lines to a `before` block to make it more DRY